### ISSUE TITLE
Concept is shown as step in autocomplete

### ISSUE THREAD

**Issue by kashishm** [getgauge/gauge-visualstudio#59]
```
# concept1
* A context step which gets executed before every scenario
```
In Autocomplete `concept1` is shown as `Step`

**Comment by apoorvam** [comment]
This works fine on MS VS 2013. Concepts are marked as `Concept` and steps as `Step`.

**Comment by sriv** [comment]
I can replicate this, happens when a concept is defined as well as used.

Concept and Step usages are identical, and autocomplete reads the concept usage and marks it as a step.

**Comment by sriv** [comment]
Fixed via 4e6a21b
